tracer/test/snapshots/CoverageRewriteTests.Rewritten.verified.txt

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using System.Threading.Tasks;
+using Datadog.Trace.Ci.Coverage;
+using Datadog.Trace.Ci.Coverage.Attributes;
+
+[assembly: CompilationRelaxations(8)]
+[assembly: RuntimeCompatibility(WrapNonExceptionThrows = true)]
+[assembly: Debuggable(DebuggableAttribute.DebuggingModes.Default | DebuggableAttribute.DebuggingModes.DisableOptimizations | DebuggableAttribute.DebuggingModes.IgnoreSymbolStoreSequencePoints | DebuggableAttribute.DebuggingModes.EnableEditAndContinue)]
+[assembly: TargetFramework(".NETStandard,Version=v2.0", FrameworkDisplayName = "")]
+[assembly: AssemblyCompany("CoverageRewriterAssembly")]
+[assembly: AssemblyConfiguration("Debug")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyInformationalVersion("1.0.0")]
+[assembly: AssemblyProduct("CoverageRewriterAssembly")]
+[assembly: AssemblyTitle("CoverageRewriterAssembly")]
+[assembly: CoveredAssembly]
+[assembly: AssemblyVersion("1.0.0.0")]
+namespace CoverageRewriterAssembly
+{
+	public class Class1
+	{
+		public string MyProperty
+		{
+			[CompilerGenerated]
+			get
+			{
+				if (!CoverageReporter.TryGetScope("/Users/tony.redondo/repos/CoverageRewriterAssembly/CoverageRewriterAssembly/Class1.cs", out var scope))
```

Comment:
This is amazing, should we do snapshot testing for the generated IL of our other Tracer integrations?